modules/sync/syncs/httpsynchronization.cpp

```diff
@@ -219,33 +227,57 @@ bool HttpSynchronization::trySyncFromUrl(std::string listUrl) {
             ghoul::filesystem::FileSystem::PathSeparator +
             filename + TempSuffix;
 
+        if (sizeData.find(line) != sizeData.end()) {
+            LWARNING(
+                fmt::format("{}: Duplicate entries: {}", _identifier, line)
+            );
+            continue;
+        }
+
         downloads.push_back(std::make_unique<AsyncHttpFileDownload>(
```

Comment:
[i know it's not your code, but since you are already in there]

```
downloads.push_back(std::make_unique<AsyncHttpFileDownload>(
    line,
    fileDestination,
    HttpFileDownload::Overwrite::Yes
));
```